api/proto/teleport/scopes/access/v1/assignment.proto

```diff
@@ -14,11 +14,11 @@
 
 syntax = "proto3";
 
-package teleport.scopedrole.v1;
+package teleport.scopes.access.v1;
 
 import "teleport/header/v1/metadata.proto";
 
-option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/scopedrole/v1;scopedrole";
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/scopes/access/v1;access";
```

Comment:
We usually try to do what [buf managed mode](https://buf.build/docs/generate/managed-mode/#go) does for `go_package`.
```suggestion
option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/scopes/access/v1;accessv1";
```